app/presenters/worldwide_organisation_presenter.rb

```diff
@@ -136,6 +147,18 @@ def sponsoring_organisations
 
 private
 
+  def contact_for_office(office_content_id)
+    contact_mapping = content_item.dig("details", "office_contact_associations").select { |office_contact_association|
+      office_contact_association["office_content_id"] == office_content_id
+    }.first
+
+    return unless contact_mapping
```

Comment:
I notice that we don't perform this check in [`roles_for_person`](https://github.com/alphagov/government-frontend/blob/c3b86d9de7be84c86ef8ba86ea759450e604a01d/app/presenters/worldwide_organisation_presenter.rb#L157). Could you possibly link me to the schema change in your publishing API PR that determines this difference?

Reply:
It's possible to have an office without a contact, whereas all the people will have a role (as we're only including those people in the content item).

Reply:
Ah nice that makes sense, thanks